app/javascript/router/Home.vue

```diff
@@ -33,7 +33,7 @@
       </div>
       <body class="flex items-center justify-center">
         <div class="container">
-          <CustomTable :data="paymentsHistory" :columns="tableColumns">
+          <CustomTable :data="paymentsHistory.slice().reverse()" :columns="tableColumns">
```

Comment:
pregunto por curiosidad nomas, por que es necesario el .slice() ?

Reply:
porque es necesario crear un nuevo arreglo pero que esté al revés. Si no le pongo .slice() parece que no se crea y sale este error:
[Vue warn]: You may have an infinite update loop in a component render function.